app/assets/stylesheets/finder_frontend.scss

```diff
@@ -25,6 +25,12 @@
   }
 }
 
+mark {
+  background-color: inherit;
+  color: inherit;
+  font-weight: bold;
```

Comment:
```suggestion
  @include govuk-typography-weight-bold;
```
to use the mixin from govuk-frontend

Reply:
Latest commit has this change made, thanks for the help!

Reply:
still seeing font-weight: bold though